.github/ISSUE_TEMPLATE/implementation-ticket.yml

```diff
@@ -0,0 +1,66 @@
+name: 🛠️ Implementation
+description: This is an implementation ticket intended for use by the maintainers of `dbt-adapters`
+title: "[<project>] <title>"
+labels: ["user docs","refinement"]
+body:
+  - type: markdown
+    attributes:
+      value: This is an implementation ticket intended for use by the maintainers of `dbt-adapters`
+
+  - type: checkboxes
+    attributes:
+      label: Housekeeping
```

Comment:
I removed the blocking issues item since we do not have that field in our project.
I moved backporting up to here since we have labels for it. We also use enhancement/bug/regression labels, so I added that item.

Reply:
for bugs we use the bug template, Core only uses this for feature work.

Reply:
Ah, great. I'll update this to be enhancement-focused then, which removes some of the either/or steps. And I'll look at the bug template and add that to this PR.